### PR TITLE
Navbar dropdown

### PR DESCRIPTION
**Pull request by madaian**
Combines the user profile and logout links under a shared dropdown

### CODE REVIEW

**Comment by Plustest** [comment]
Can one of the admins verify this patch?

File: templates/base.html [hunk 1]
```diff
@@ -88,17 +88,21 @@
...
-                                    <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
-                                    {{ user.username }}
-                                </a>
-                            </li>
-                            <li role="presentation">
-                                <a href="{% url 'logout' %}">
+                            <li class="dropdown">
+                              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
+                                {{ user.first_name }} {{ user.last_name }} <span class="caret"></span>
```
Comment (raphendyr) [inline templates/base.html]: Maybe one of these would be cleaner: `user.get_full_name` (in django AbstractUser) or `user.userprofile.shortname` (in a-plus/userprofile/models.py).

Comment (raphendyr) [reply templates/base.html]: rofl.. the `shortname` is currently used only in some tests... so maybe go with the former.

Comment (raphendyr) [reply templates/base.html]: I don't know what I'm thinking. It's probably better to have this here. so I'll just merge this.

**Pull request merged**